File: snapshots/output-net10.0/syntax/Main/Structs.cs
  using System.Diagnostics.CodeAnalysis;
//      ^^^^^^ read_access scip-dotnet nuget . . System/
//             enclosing_range 0:6-0:24
//      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ import scip-dotnet nuget . . CodeAnalysis/
//                                      enclosing_range 0:6-0:37
//             ^^^^^^^^^^^ read_access scip-dotnet nuget . . Diagnostics/
//                         enclosing_range 0:6-0:24
//                         ^^^^^^^^^^^^ read_access scip-dotnet nuget . . CodeAnalysis/
//                                      enclosing_range 0:6-0:37

  namespace Main;
//          ^^^^ read_access scip-dotnet nuget . . Main/
//               enclosing_range 2:10-2:14

  [SuppressMessage("ReSharper", "all")]
// ^^^^^^^^^^^^^^^ read_access scip-dotnet nuget System.Runtime 10.0.0.0 CodeAnalysis/SuppressMessageAttribute#`.ctor`().
//                 enclosing_range 4:1-4:16
  public class Structs
//             ^^^^^^^ definition scip-dotnet nuget . . Main/Structs#
//                     enclosing_range 4:0-16:1
//                     documentation ```cs\nclass Structs\n```
//                     kind Class
  {
      struct BasicStruct
//           ^^^^^^^^^^^ definition scip-dotnet nuget . . Main/Structs#BasicStruct#
//                       enclosing_range 7:4-15:5
//                       documentation ```cs\nstruct BasicStruct\n```
//                       kind Struct
      {
          public int Property1;
//                   ^^^^^^^^^ definition scip-dotnet nuget . . Main/Structs#BasicStruct#Property1.
//                             enclosing_range 9:8-9:29
//                             documentation ```cs\npublic int BasicStruct.Property1\n```
//                             kind Field
//                             relationship type_definition scip-dotnet nuget System.Runtime 10.0.0.0 System/Int32#

          public BasicStruct(int field1)
//               ^^^^^^^^^^^ definition scip-dotnet nuget . . Main/Structs#BasicStruct#`.ctor`().
//                           enclosing_range 11:8-14:9
//                           documentation ```cs\npublic BasicStruct.BasicStruct(int field1)\n```
//                           kind Constructor
//                               ^^^^^^ definition scip-dotnet nuget . . Main/Structs#BasicStruct#`.ctor`().(field1)
//                                      enclosing_range 11:8-14:9
//                                      documentation ```cs\nint field1\n```
//                                      kind Parameter
//                                      relationship type_definition scip-dotnet nuget System.Runtime 10.0.0.0 System/Int32#
          {
              Property1 = field1;
//            ^^^^^^^^^ write_access scip-dotnet nuget . . Main/Structs#BasicStruct#Property1.
//                      enclosing_range 13:12-13:30
//                        ^^^^^^ read_access scip-dotnet nuget . . Main/Structs#BasicStruct#`.ctor`().(field1)
//                               enclosing_range 13:12-13:30
          }
      }
  }

File: snapshots/output-net10.0/syntax/VBMain/Structs.vb
  Imports System.Diagnostics.CodeAnalysis
'         ^^^^^^ read_access scip-dotnet nuget . . System/
'                enclosing_range 0:0-0:39
'         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ import scip-dotnet nuget . . CodeAnalysis/
'                                         enclosing_range 0:0-0:39
'                ^^^^^^^^^^^ read_access scip-dotnet nuget . . Diagnostics/
'                            enclosing_range 0:0-0:39
'                            ^^^^^^^^^^^^ read_access scip-dotnet nuget . . CodeAnalysis/
'                                         enclosing_range 0:0-0:39

  Namespace VBMain
'           ^^^^^^ read_access scip-dotnet nuget . . VBMain/
'                  enclosing_range 2:0-2:16
      <SuppressMessage("ReSharper", "all")>
'      ^^^^^^^^^^^^^^^ read_access scip-dotnet nuget System.Runtime 10.0.0.0 CodeAnalysis/SuppressMessageAttribute#`.ctor`().
'                      enclosing_range 3:4-4:24
      Public Class Structs
'                  ^^^^^^^ definition scip-dotnet nuget . . VBMain/Structs#
'                          enclosing_range 3:4-12:13
'                          documentation ```vb\nClass Structs\n```
'                          kind Class
          Structure BasicStruct
'                   ^^^^^^^^^^^ definition scip-dotnet nuget . . VBMain/Structs#BasicStruct#
'                               enclosing_range 5:8-11:21
'                               documentation ```vb\nStructure BasicStruct\n```
'                               kind Struct
              Public Property1 As Integer
'                    ^^^^^^^^^ definition scip-dotnet nuget . . VBMain/Structs#BasicStruct#Property1.
'                              enclosing_range 6:12-6:39
'                              documentation ```vb\nPublic BasicStruct.Property1 As Integer\n```
'                              kind Field
'                              relationship type_definition scip-dotnet nuget System.Runtime 10.0.0.0 System/Int32#

              Public Sub New(ByVal field1 As Integer)
'                        ^^^ definition scip-dotnet nuget . . VBMain/Structs#BasicStruct#`.ctor`(+1).
'                            enclosing_range 8:12-10:19
'                            documentation ```vb\nPublic Sub BasicStruct.New(field1 As Integer)\n```
'                            kind Constructor
'                                  ^^^^^^ definition scip-dotnet nuget . . VBMain/Structs#BasicStruct#`.ctor`(+1).(field1)
'                                         enclosing_range 8:12-8:51
'                                         documentation ```vb\nfield1 As Integer\n```
'                                         kind Parameter
'                                         relationship type_definition scip-dotnet nuget System.Runtime 10.0.0.0 System/Int32#
                  Property1 = field1
'                 ^^^^^^^^^ write_access scip-dotnet nuget . . VBMain/Structs#BasicStruct#Property1.
'                           enclosing_range 8:12-10:19
'                             ^^^^^^ read_access scip-dotnet nuget . . VBMain/Structs#BasicStruct#`.ctor`(+1).(field1)
'                                    enclosing_range 8:12-10:19
              End Sub
          End Structure
      End Class
  End Namespace
